### PR TITLE
Remove ONNX code in DLK directory

### PR DESCRIPTION
**Pull request by iizukak**
## Motivation and Context
Implement #407 .
I just simply onnx code.

## How has this been tested?

- [x] unit testing
- [x] `git grep -i onnx` return no line.
- [x] `find ./ -name '*onnx*'`

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature / Optimization (non-breaking change which adds functionality)
- [ ]  Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by iizukak** [comment]
@nlpng Please check if I remove needed file or not.

**Comment by tkng** [comment]
Readability Approval

**Comment by tkng** [comment]
BTW, nice PR!

**Comment by iizukak** [comment]
@nlpng [WIP] removed. want your review :)

File: dlk/tests/onnx/test_onnx_io.py [hunk 1]
```diff
@@ -1,108 +0,0 @@
...
-        """Test code for importing lmnet classification via ONNX."""
-        # Current lmnet classification onnx has several problems,
-        # so we leave this test for the near future.
-        from frontend.onnx import OnnxIO
-
-        onnx_path = path.join('examples',
-                              'classification',
-                              'lmnet_quantize_cifar10_stride_2.20180523.3x3',
-                              # 'lmnet_quantize_cifar10_max_pooling',
-                              'onnx_minimal_graph_with_shape.pb')
```
Comment (nlpng) [inline dlk/tests/onnx/test_onnx_io.py]: @iizukak Can we also remove the test `.pb` file?

Comment (iizukak) [reply dlk/tests/onnx/test_onnx_io.py]: @nlpng Oh, we should remove. thanks.

Comment (iizukak) [reply dlk/tests/onnx/test_onnx_io.py]: @nlpng Removed at fd306bba60ba1ac9f1f4f4c2e776f44def2efedd

**Pull request merged**